tests/smoke/smoketests.sh

```diff
@@ -215,6 +215,34 @@ for BACKEND in ${BACKENDS[@]}; do
 			fi
 			echo " "
 			
+			NTEST=256
+			if [ -f "${TEST_BIN_DIR}/gmres_${BACKEND}" ]
+			then
+			    echo ">>>      [x]           [ ]       Testing the GMRES real algorithm for the random generated"
+			    echo "                                 matrix (${NTEST}x${NTEST}) with preconditioner. This test"
+			    echo "                                 verifies against a predifined solution vector. The test"
+			    echo "                                 employs the grb::Launcher in automatic mode. It uses"
+			    echo "                                 direct-mode file IO."
+			    $runner ${TEST_BIN_DIR}/gmres_${BACKEND} --n ${NTEST}  &> ${TEST_OUT_DIR}/gmres_${BACKEND}_${P}_${T}.log
+			    head -1 ${TEST_OUT_DIR}/gmres_${BACKEND}_${P}_${T}.log
+			    grep 'Test OK' ${TEST_OUT_DIR}/gmres_${BACKEND}_${P}_${T}.log || echo "Test FAILED"
+			    echo " "
+			fi
+
+			NTEST=256
```

Comment:
Does it makes sense to make this second one a non-power-of-two and (much) larger to trigger any potential issues in parallelisation?

Reply:
We can probably use any number here, or we have a bug. I will increase it to 50000.